cluster_deploy/pulsar/update_helm.sh

```diff
@@ -116,7 +116,7 @@ function updatePulsarHelmChart() {
    debugMsg "tgtRelVer=${tgtRelVer}"
 
    # Update Pulsar cluster name
-   sed -i "s/fullnameOverride:.*/fullnameOverride: ${clstrName}/g" $1 
+   sed -i '' "s/fullnameOverride:.*/fullnameOverride: ${clstrName}/g" $1 
```

Comment:
I think this fix is Mac only. It will break on Linux machines. Have you tested on Linux? 
As to the fix, I think we can do something as below. Open for discussion though.
1) emphasize the GNU sed is required, or
2) add a code branch to make difference between Linux and Mac